packages/devextreme/js/__internal/viz/chart_components/m_base_chart.ts

```diff
@@ -181,13 +194,11 @@ function sortRollingStocksByValue(rollingStocks) {
   return positiveRollingStocks.concat(negativeRollingStocks);
 }
 
-function prepareOverlapStacks(rollingStocks) {
-  let i;
-  let currentRollingStock;
+function prepareOverlapStacks(rollingStocks): void {
   let root;
 
-  for (i = 0; i < rollingStocks.length - 1; i++) {
-    currentRollingStock = root || rollingStocks[i];
+  for (let i = 0; i < rollingStocks.length - 1; i += 1) {
```

Comment:
just wondering, whats the point of using += 1 instead of ++?

Reply:
new [lint rule](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/__internal/.eslintrc.js#L119) for this